app/jobs/scrape_job.rb

```diff
@@ -0,0 +1,66 @@
+class PostScrapeWorker < ActiveJob::Base
+  queue_as :default
+
+  require 'capybara'
+  require 'capybara/poltergeist'
+
+  include Sidekiq::Worker
+  include Capybara::DSL
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, timeout: 60, js_errors: false)
+  end
+  Capybara.default_driver = :poltergeist
+
+  def perform(new_catalogue)
+    new_catalogue.each do |catalogue_num|
+      begin
+
+        #visit page
+        visit "http://salefinder.com.au/woolworths-catalogue/weekly-specials-catalogue/#{catalogue_num}/catalogue"
+        
+        #creating new catalogue
+        @catalogue = Catalogue.new

```

Comment:
No need for `@catalogue` to be an instance variable. Try to minimize variable scope where possible.